src/main/java/org/mamute/list/TabsHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.mamute.list;
+
+import br.com.caelum.vraptor.http.route.Router;
+import net.vidageek.mirror.dsl.Mirror;
+import org.apache.commons.lang.WordUtils;
+import org.mamute.controllers.ListController;
+
+import javax.inject.Inject;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class TabsHelper {
+
+	private final Router router;
+
+	@Inject
+	public TabsHelper(Router router) {
+		this.router = router;
+	}
+
+	public Tab tabForType(Tab.Type tabType) {
+		String localizationKey = "menu.top." + tabType.getStringValue();
+		String methodName = "top" + WordUtils.capitalize(tabType.getStringValue());
+		return new Tab(tabType, localizationKey, urlForMethod(methodName));
```

Comment:
I considered this actually. What I didn't want to do is create a DOM with a lot of horizontal dependencies (e.g. the router dependency). Figuring out a controller method responsible for specific tag  in the tag itself feels even more scary as this introduces upwards dependency between low level DOM and controller. It's your repo, so I can merge the Helper and Tab if you insist, but if feels wrong?